docs/docs/demos/simple-select.md

```diff
@@ -1,7 +1,7 @@
 # Simple Data Selection
 
 This demo is the simplest one in our set that still serves to
-demonstrate the basics of DBSP.  Start by firing up the DBSP demo and
+demonstrate the basics of Feldera.  Start by firing up the Feldera demo and
```

Comment:
first instance cannot be right due to the same reasons. Perhaps "Feldera software".